From 83e9444e3f8578beb7de65c5a6b463d4e4e59c2b Mon Sep 17 00:00:00 2001
From: fu-shanqing <fushanqing@kylinos.com>
Date: Fri, 23 Sep 2022 11:29:42 +0800
Subject: [PATCH] use-installed-packages-instead-of-dnf-downloading

---
 src/supermin.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/supermin.ml b/src/supermin.ml
index 9f838d9..7bd7de8 100644
--- a/src/supermin.ml
+++ b/src/supermin.ml
@@ -92,7 +92,7 @@ let main () =
     let mode = ref None in
     let outputdir = ref "" in
     let packager_config = ref "" in
-    let use_installed = ref false in
+    let use_installed = ref true in
     let size = ref None in
     let include_packagelist = ref false in
 
-- 
2.30.0

