From bb6a164d258c70d69b74e946f9e01d1503d66df0 Mon Sep 17 00:00:00 2001
From: maminjie <maminjie1@huawei.com>
Date: Tue, 15 Dec 2020 16:39:20 +0800
Subject: [PATCH] use installed packages instead of dnf downloading

---
 src/supermin.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/supermin.ml b/src/supermin.ml
index f9798f9..7e4bd6c 100644
--- a/src/supermin.ml
+++ b/src/supermin.ml
@@ -91,7 +91,7 @@ let main () =
     let mode = ref None in
     let outputdir = ref "" in
     let packager_config = ref "" in
-    let use_installed = ref false in
+    let use_installed = ref true in
     let size = ref None in
     let include_packagelist = ref false in
 
-- 
2.23.0

