From a5d2b966fe89e66b7e5f0bb5387f4f0c855cfb54 Mon Sep 17 00:00:00 2001
From: wangqing <wangqing@uniontech.com>
Date: Fri, 8 Oct 2021 16:20:26 +0800
Subject: [PATCH] fix cannot detect package manager on UnionTech

Signed-off-by: wangqing <wangqing@uniontech.com>

diff --git a/src/ph_rpm.ml b/src/ph_rpm.ml
index 61537ce..a0d3405 100644
--- a/src/ph_rpm.ml
+++ b/src/ph_rpm.ml
@@ -31,10 +31,11 @@ let stringset_of_list pkgs =
 let fedora_detect () =
   Config.rpm <> "no" && Config.rpm2cpio <> "no" && rpm_is_available () &&
     (Config.yumdownloader <> "no" || Config.dnf <> "no") &&
-    (List.mem (Os_release.get_id ()) [ "fedora"; "rhel"; "centos"; "openEuler" ] ||
+    (List.mem (Os_release.get_id ()) [ "fedora"; "rhel"; "centos"; "openEuler"; "uos" ] ||
      try
        (stat "/etc/redhat-release").st_kind = S_REG ||
        (stat "/etc/fedora-release").st_kind = S_REG ||
+       (stat "/etc/UnionTech-release").st_kind = S_REG ||
        (stat "/etc/openEuler-release").st_kind = S_REG
      with Unix_error _ -> false)
 
-- 
2.20.1

