From 1b43ce5da29fa5a1b9b405e78808f9e43b30d734 Mon Sep 17 00:00:00 2001
From: fu-shanqing <fushanqing@kylinos.com>
Date: Fri, 23 Sep 2022 11:37:04 +0800
Subject: [PATCH] add-pie-and-bind_now-flags

---
 src/Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile.in b/src/Makefile.in
index c6e108c..28d8b18 100644
--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -485,7 +485,7 @@ supermin_DEPENDENCIES = $(OBJECTS)
 supermin_LINK = \
 	./supermin-link.sh \
 	  $(OCAMLFIND) $(BEST) $(OCAMLFLAGS) $(OCAMLPACKAGES) \
-	  $(OBJECTS) -o $@
+	  $(OBJECTS) -cclib "-pie -Wl,-z,now" -o $@
 
 BUILT_SOURCES = format-ext2-init-bin.h
 SUFFIXES = .cmo .cmi .cmx .ml .mli .mll .mly
-- 
2.30.0

